pages/dkp/kaptain/1.2.0/sdk/private-registries/index.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: layout.pug
+navigationTitle: Accessing a Private Docker Registry with Self-Signed Certificates
+title: Configuring Access to a Private Docker Registry with Self-Signed Certificates
+beta: false
+menuWeight: 2
+---
+
+This guide focuses on configuring access to private Docker registry with self-signed certificates and is based on
+`configuration-and-credentials` guide. It focuses on:
+
+* secure automatic mounting of a registry certificate to
```

Comment:
These 2 bullet points don't start with capital letters, is this a problem with consistency?

Reply:
My understanding is that if a list is introduced by a complete sentence, each bullet point needs to begin with a capital letter. This is not the case here as it's more like a single sentence split into bullets for readability. But I'd defer this question to @cmastr.

Reply:
consistency is key with bullets, but the rest is up to the writer. This is fine as-is.